vp100_ros2_node:
  ros__parameters:
    serialport_name: "/dev/nvilidar"
    serialport_baud: 115200
    frame_id: "laser_frame"
    resolution_fixed: false
    auto_reconnect: true
    reversion: false
    inverted: false
    angle_min: -180.0
    angle_max: 180.0
    range_min: 0.001
    range_max: 64.0
    aim_speed: 6.0
    sampling_rate: 3
    angle_offset_change_flag: false
    angle_offset: 0.0
    ignore_array_string: ""
    filter_sliding_enable: true
    filter_tail_enable: true
    filter_sliding_jump_threshold: 50
    filter_sliding_max_range_flag: false
    filter_sliding_max_range: 8000
    filter_sliding_window: 3
    filter_tail_distance_limit_flag: false
    filter_tail_distance_limit_value: 8000
    filter_tail_level: 8
    filter_tail_neighbors: 0
